tracing/flags.go

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package tracing
+
+import (
+	"github.com/urfave/cli/v2"
+)
+
+var Flags = []cli.Flag{
+	// Tracing Flags
+
+	&cli.BoolFlag{
+		EnvVars: []string{"VELA_OTEL_TRACING_ENABLE"},
+		Name:    "tracing.enable",
+		Usage:   "enable otel tracing",
+		Value:   false,
+	},
+
+	// Exporter Flags
+
+	&cli.StringFlag{
+		EnvVars: []string{"VELA_OTEL_TRACING_SERVICE_NAME"},
+		Name:    "tracing.service.name",
+		Usage:   "set otel tracing service name",
+		Value:   "go-otel-server",
+	},
+	&cli.StringFlag{
+		EnvVars: []string{"VELA_OTEL_TRACING_EXPORTER_SSL_CERT_PATH"},
+		Name:    "tracing.exporter.cert_path",
+		Usage:   "set the path to certs used for communicating with the otel exporter. if not set, will use insecure communication",
+	},
+	&cli.StringFlag{
+		EnvVars: []string{"VELA_OTEL_TRACING_ENDPOINT"},
+		Name:    "tracing.exporter.endpoint",
+		Usage:   "set the otel exporter endpoint",
+		Value:   "127.0.0.1:4318",
```

Comment:
should have an empty value as the default?

Reply:
removed this in favor of `VELA_OTEL_EXPORTER_OTLP_ENDPOINT`, intentionally has `VELA_` such that it does NOT get picked up by the library, instead this URL is _parsed_, _validated_ then used in `otlp.WithEndpoint`